code/pizzabot.js

```diff
@@ -1,7 +1,193 @@
+"Great, I'll get started on your Hawaiian Pizza right away, it will cost 240 kr"
+
+*/
+/*
+
+const orderTotal = (pizzaPrice * orderQuantity)
+alert(`Great, I'll get started on your ${orderName} right away, it will cost ${orderTotal} kr`)
+
+
+/*
+## **Iteration 5 - Add conditionals**
+
+We want you to add conditionals statements (if-statements) to your program to improve it.
+
+### **Checking that the Pizza is on the menu**
+When the user enters the `orderName` prompt, add an if-statement to check if the input text matches any of the pizza name variables
+(`vegetarian`, `hawaiian`, `pepperoni`).
+If yes - show the next promt,
+if no - show an alert that says `Select a pizza from the menu` or something similar.
+
+*/
+
+/*
+### **Show cooking times**
+
+Before you print the final `Great, I'll get started on your X right away,
+it will cost Y kr` message to the user, calculate the cooking time based on these rules and add it to the message:
+
+- 1-2 pizzas: `The pizzas will take 10 minutes.`
+- 3-5 pizzas: `The pizzas will take 15 minutes.`
+- 6+ pizzas: `The pizzas will take 20 minutes.`
+
+For example, if I ordered 3 Hawaiian pizzas, the message should now read:
+"Great, I'll get started on your Hawaiian Pizza right away, it will cost 240 kr. The pizzas will take 15 minutes."
+
+*/
+
+/*
+const X = "Vegetarian Pizza"
+const Y = "Hawaiian Pizza"
+const Z = "Pepperoni Pizza"
+const pizzaPrice = 80
+
+/*
+alert(`Hey! Happy to serve your pizza. On our menu we have ${X}, ${Y} and ${Z}`)
+
+let orderName = prompt(`Enter the name of the pizza you want to order today`)
+
+if (orderName == X || orderName == Y || orderName == Z) {
+  const orderQuantity = prompt(`"How many of ${orderName} do you want?`)
+  const orderTotal = (pizzaPrice * orderQuantity)
+  if (orderQuantity < 3) {
+    alert(`The pizzas will take 10 minutes. Great, I'll get started on your ${orderName} right away, it will cost ${orderTotal} kr.`)
+  } else if (orderQuantity >= 3 && orderQuantity <= 5) {
+    alert(`The pizzas will take 15 minutes. Great, I'll get started on your ${orderName} right away, it will cost ${orderTotal} kr.`)
+  } else {
+    alert(`The pizzas will take 20 minutes. Great, I'll get started on your ${orderName} right away, it will cost ${orderTotal} kr.`)
+  }
+} else {
+  alert(`Select a pizza from the menu`)
+}
+
+
+
+/*
+
+## **Iteration 6 - Add functions**
+
+Clean up your code by moving some code into functions and then calling that function when needed.
+
+The functions should all reflect the tasks we've completed so far. So, the functions should be:
+
+1. `checkOrderName()` which should take the `orderName` variable as an argument and return `true` or `false` if the pizza is on the menu or not.
+2. `totalCost()` which takes `orderQuantity` as an argument and returns the total cost for the order.
+3. `cookingTime()` which takes `orderQuantity` and returns the number of minutes it will take to finish the order.
+
+
+*/
+
+
+const X = 'Vegetarian Pizza';
```

Comment:
The naming could be better here. The names should describe its data/value. Here's an example:
```suggestion
const vegetarian = 'Vegetarian Pizza';
```